server/command.go

```diff
@@ -17,6 +17,8 @@ const helpText = `* |/zoom start| - Start a zoom meeting`
 const oAuthHelpText = `* |/zoom connect| - Connect to zoom
 * |/zoom disconnect| - Disconnect from zoom`
 
+const followStatusHelpText = `* $/zoom follow_status [on|off] $ - Automatically update your Mattermost status based on Zoom status"`
```

Comment:
There's an extra double quote at the end of this line

```suggestion
const followStatusHelpText = `* $/zoom follow_status [on|off] $ - Automatically update your Mattermost status based on Zoom status`
```